src/index.html

```diff
@@ -21,7 +21,385 @@
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+        <div class="stars-wrapper">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <span class="stars__reviews-text">Reviews: 5</span>
+          </div>
+        </div>
+        <p class="card__price">
+          <span>Price:</span>
+          <span class="card__price-number">$2,199</span>
+        </p>
+        <a
+          href="#"
+          class="card__link"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 inch Retina display"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+        <div class="stars-wrapper">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <span class="stars__reviews-text">Reviews: 5</span>
+          </div>
+        </div>
+        <p class="card__price">
+          <span>Price:</span>
+          <span class="card__price-number">$2,199</span>
+        </p>
+        <a
+          href="#"
+          class="card__link"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 inch Retina display"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+        <div class="stars-wrapper">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <span class="stars__reviews-text">Reviews: 5</span>
+          </div>
+        </div>
+        <p class="card__price">
+          <span>Price:</span>
+          <span class="card__price-number">$2,199</span>
+        </p>
+        <a
+          href="#"
+          class="card__link"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The product information is repeated across multiple card elements. If this is intentional, ensure that it aligns with the project requirements. Otherwise, consider varying the content or dynamically generating these elements if applicable.